pyrdfj2/sparqlbuilder.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from pathlib import Path
+
+from jinja2 import Environment, FileSystemLoader, meta
+
+from pyrdfj2.j2_functions import Filters, Functions
+from pyrdfj2.pyrdfj2 import SparqlBuilder
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_TEMPLATES_FOLDER = Path(__file__).parent.absolute() / "templates"
+
+
+class J2SparqlBuilder(SparqlBuilder):
```

Comment:
Given the fact we want to use this in a more generic way in other context I strongly suggest renaming.

in `pykg2tbl` we strictly use this for generating `sparql`
but in `pysubyt`we aim producing mainly `ttl` files 

suggestion `J2RDFSyntaxBuilder`  and equivalent for its base class